README.md

```diff
@@ -7,7 +7,8 @@
 Describe the differences between a SQL and NoSQL database, and when you might use each.
 
 ```text
-Your answer...
+SQL databases are relational in nature and are rigid in how it is structured. There are clearly defined relationships, and each table (and SQL DBs use tables) has specific architecture to its data.  SQL Databases can be used in pretty much any project, although it isn't always the idea solution.  It's probably the right choice when the data is highly structured and that structure is generally unchanging.  It's also probably a good idea if you want separation of data, as there are multiple tables.
+NoSQL databases, on the other hand, are non-relational, and have all the data (as objects...or documents) lumped together in one location.  The schema is highly fluid, and there is little to no rigidity and the data you can put into it.  This allows for faster searching, and often times one query will do when in a relational database it would take many queries and joins.  This type of database is well suited for a project that has fluid requirements.  Things can change on a dime, and so the database can handle those changes.  It's also scalable as changes in a NoSQL database are no problem, whereas the rigid structure of a SQL database would create cascading effect with single changes that might cause difficulty.
 ```
 
```

Comment:
Nice.

MongoDB's indexing is a big part of its speed. There's a lot of fancy algorithms and leveraging of supplementary data structures making mongo performant and scalable.

If you're interested:
https://docs.mongodb.com/manual/indexes/